content/millennium/r4/financial/account.md

```diff
@@ -64,7 +66,7 @@ Search for Accounts that meet supplied query parameters:
 
 _Implementation Notes_
 
-* The Account balance extension is only returned on statement and guarantor-balance types
+* The Account balance extension is only returned on statement, guarantor-balance, and insurance-benefit types
```

Comment:
I'm wondering if we want to update this line further to include the hotlink to the `#custom-extensions` that lists the `[Account Balance]` here. Thoughts?

Reply:
Updated - https://github.com/cerner/fhir.cerner.com/pull/664/commits/0a140bceb0871cf29cb01a44e0889381ff16109f#diff-c8eb47813f9124e188b80dbfe03423839685e5b0f517fa4bcf1a8222a4db0570R69

Screenshots are also updated above.

Reply:
+1 to changes.